apps/backend/test/habit.test.ts

```diff
@@ -105,6 +105,23 @@ describe("/habits", () => {
       expect(res.body).toStrictEqual(JSON.parse(JSON.stringify(habit)));
     });
   });
+
+  describe("PUT", () => {
+    test("Edit a habit", async () => {
+      prismaMock.habit.findFirst.mockResolvedValue(habit);
+      const newChanges = {
```

Comment:
Probably want to mock `prisma.habit.update` as well

Reply:
added the new Prisma mock, nice catch.